src/app/main/main.html

```diff
@@ -66,4 +72,42 @@
     </bookmark-list>
   </modal>
   <dataset-modal></dataset-modal>
+  <tour step="currentStep">
+    <virtual-step
```

Comment:
I'm not opposed to it. suggestions for where to put the `.html` ? I could create a new sub-directory under `components` to store this and the scss? I could just add it to assets?